### DISCUSSION TITLE
Has anyone else encountered this issue?

### DISCUSSION THREAD

**Discussion by LWFlouisa** [FreeTubeApp/FreeTube#6034] [Q&A]
![PublishedIn1969](https://github.com/user-attachments/assets/ac948063-1067-4253-9b2b-2c0cfa1ebf6e)

I wasn't aware Youtube had been in business that long!

**Comment by absidue** [comment]
When both API backends fails the entire page breaks because there is no information to show, so that timestamp just falls back to the default value of start of unix time which is 1970-01-01 00:00:00 UTC, so yes when FreeTube converts it to your timezone, it can go even further back.
